amgut/lib/data_access/ag_data_access.py

```diff
@@ -580,12 +578,29 @@ def getBarcodesByKit(self, kitid):
         return [row[0] for row in results]
 
     def checkPrintResults(self, kit_id):
+        """Checks whether or not results are available for a given `kit_id`
+
+        Parameters
+        ----------
+        kit_id : str
+            The supplied kit identifier to check for results availability.
+
+        Returns
+        -------
+        bool
+            Whether or not the results are ready for the supplied kit_id.
+
+        Notes
+        -----
+        If a `kit_id` does not exist this function will return False, as no
+        results would be available for a non-existent `kit_id`.
+        """
         sql = "SELECT print_results FROM ag_handout_kits WHERE kit_id = %s"
         results = self._sql.execute_fetchone(sql, [kit_id])
         if results is None:
-            return None
+            return False
         else:
-            return results[0].strip()
+            return results[0]
```

Comment:
It is guaranteed to be a Python bool, because the column is set to be boolean and psycopg deals wit this nicely so no need to preprocess the output before coercing it into a recognizable type.